clienttable.sql

```diff
@@ -0,0 +1,41 @@
+USE JCAC_CormacF;
+DELIMITER $$
+DROP PROCEDURE IF EXISTS my_client_table $$
+CREATE PROCEDURE my_client_table()
```

Comment:
Again, discuss this with your team. You need to at least be consistent with the rest of the project when naming your procedures!